miden-tx/tests/integration/main.rs

```diff
@@ -72,14 +72,16 @@ pub fn get_account_with_default_account_code(
     public_key: Word,
     assets: Option<Asset>,
 ) -> Account {
+    use std::collections::BTreeMap;
```

Comment:
I don't think it makes a difference for tests but just curious as to why sometimes we import `BTreeMap` from `std` and why sometimes from `alloc`

Reply:
Good question, I automatically imported those. Changed it